pages/wiki/watchfaces-creation.md

```diff
@@ -77,6 +78,30 @@ layout: documentation
     }</code></pre>
   <p>The UI slider then acts as a controller to see how the watchface reacts to the different values. Note that the slider gives a real value from 0.0 to 1.0. To simulate the integer 0 to 100 provided by the real battery on the watch, we scale and convert to a fixed value in the code above.</p>  
 </div>
+<div class="page-header">
+  <h1 id="how-to-share">Share your watchface</h1>
+</div>
+<div>
+  <p>Pull requesting your finished watchface to the <a href="https://github.com/AsteroidOS/unofficial-watchfaces">unofficial-watchfaces repo</a> is always very welcome! Follow this guide to ensure consitency with existing community watchfaces and a quick review process.</p>
+  <p>Edit the <code>README.md</code> and sort your watchface entry into the list alphabetically. The required thumbnails can be conveniently created by using the <code>./test-in-qmlscene.sh</code> script. </p>
+  <p>Please mind to give licensing information for all licensed material used in your watchface, at the bottom section of the <code>README.md</code>. Only works issued under an open license that allows embedded redistribution (OFL/SIL, Apache, BSD, CC-BY, etc.), are suitable to be merged into our repositories.</p> 
```

Comment:
`Please mind to give licensing information for all licensed material used in your watchface`

This sounds like a bit of a weird sentence, I came up with two other ones that in my opinion sound better:
- Remember to provide licensing information to all licensed material used in your watchface
- Don't forget to provide licensing information for all licensed material used in your watchface